src/pivot-table/hooks/use-scroll-direction.ts

```diff
@@ -0,0 +1,53 @@
+import { useCallback, useRef } from "react";
+
+interface OnScroll {
+  scrollLeft: number;
+  scrollTop: number;
+}
+
+export enum ScrollDirection {
+  Forward = "Forward",
+  Back = "Back",
+  None = "None", // If the user is not scrolling in any direction, use this value
+}
+
+const useScrollDirection = () => {
+  const prevScrollLeft = useRef(0);
+  const prevScrollTop = useRef(0);
+  const verticalScrollDirection = useRef(ScrollDirection.None);
+  const horizontalScrollDirection = useRef(ScrollDirection.None);
+
+  const scrollHandler = useCallback(
+    ({ scrollTop, scrollLeft }: OnScroll) => {
+      // Set vertical scroll direction
+      if (scrollTop > prevScrollTop.current) {
+        verticalScrollDirection.current = ScrollDirection.Forward;
+      } else if (scrollTop < prevScrollTop.current) {
+        verticalScrollDirection.current = ScrollDirection.Back;
+      } else {
+        verticalScrollDirection.current = ScrollDirection.None;
+      }
+
+      // Set horizontal scroll direction
+      if (scrollLeft > prevScrollLeft.current) {
+        horizontalScrollDirection.current = ScrollDirection.Forward;
+      } else if (scrollLeft < prevScrollLeft.current) {
+        horizontalScrollDirection.current = ScrollDirection.Back;
+      } else {
+        horizontalScrollDirection.current = ScrollDirection.None;
+      }
```

Comment:
I think you dont need to calculate the direction, it already gives it to you as `onScroll` argument:
<img width="1356" alt="Screenshot 2023-11-22 at 10 24 34" src="https://github.com/qlik-oss/sn-pivot-table/assets/29652890/602232aa-d918-4abe-93fe-26d3a7acd6fb">

I logged it an find this: 
<img width="354" alt="Screenshot 2023-11-22 at 10 25 23" src="https://github.com/qlik-oss/sn-pivot-table/assets/29652890/e2d0d467-ee48-4449-bc5f-a9e082709612">

(Maybe) is there any other reason that you did not consume these arguments directly?